js/src/layers/LayerGroup.js

```diff
@@ -10,6 +10,10 @@ export class LeafletLayerGroupModel extends layer.LeafletLayerModel {
     return {
       _view_name: 'LeafletLayerGroupView',
```

Comment:
Sorry for not looking at this more deeply the other time. This is actually missing a call to the `super`'s `defaults` method:

```javascript
...super.defaults(),
```

Similar to here: https://github.com/jupyter-widgets/ipyleaflet/blob/master/js/src/layers/AntPath.js#L11

This will merge the superclass defaults attribute which contains the proper module names and versions. 

`LayerGroup` is actually not the only one missing this, `ImageOverlay` also misses it (and maybe others).